pmctl

```diff
@@ -52,6 +61,8 @@ start_nginx() {
     echo "*** START NGINX ***"
     if [[ ! -n "$(docker-compose ps | grep nginx | grep Up)" ]]; then
         echo "*** INFO: nginx is not running, waiting ***"
```

Comment:
echo message could be changed to replacing ```waiting``` with ```starting``` to align with the action better.